ibis/backends/dask/tests/execution/test_operations.py

```diff
@@ -128,15 +124,16 @@ def test_aggregation_group_by(t, df, where, ibis_func, dask_func):
     )
     result = expr.execute()
 
-    dask_where = where(df)
+    dask_where = where(df.compute())
```

Comment:
why this change?

Reply:
Because the where clause will not work directly against a dask frame